src/Microsoft.VisualStudio.Extensibility.Testing.Xunit.Shared/Harness/VisualStudioInstanceFactory.cs

```diff
@@ -179,43 +181,15 @@ private async Task UpdateCurrentlyRunningInstanceAsync(Version version, string?
             }
         }
 
-        internal static IEnumerable<Tuple<string, Version, ImmutableHashSet<string>, InstanceState>> EnumerateVisualStudioInstances()
+        internal static IEnumerable<Tuple<string, Version, ImmutableHashSet<string>, InstanceState, string>> EnumerateVisualStudioInstances()
         {
-            foreach (var result in EnumerateVisualStudioInstancesInRegistry())
```

Comment:
Note that removing this code is probably explicitly killing support for anything not using our current installer. Given we moved to it in VS2017 and that's our oldest "still in support" VS...this is fine. But we can probably make a note to pull additional support at some point too.